system-test/spanner.ts

```diff
@@ -1020,6 +1009,7 @@ describe('Spanner', () => {
       // Create a second database since only one pending backup can be created
       // per database.
       database2 = instance.database(generateName('database'));
+      RESOURCES_TO_CLEAN.push(database2);
```

Comment:
Could you move these pushes to `RESOURCES_TO_CLEAN` to after the creation operation resolves? I liked how you did that in the global before hook. It makes sense to continue the trend.